### ISSUE TITLE
Feature request: Slack integration

### ISSUE THREAD

**Issue by sacarino** [dei79/azure-status-page#2]

**Comment by dei79** [comment]
Yeah Slack would be great to have or better we are shooting for Zapier integration to be more flexible. With Zapier Slack support would be a breeze.

**Comment by dbsanfte** [comment]
Any update on this?

**Comment by charry223** [comment]
Thanks for sharing. Guys if these keys don't work for you and you don't mind spending a little money, then you can visit vanskeys.com to get heap and genuine keys for office in all versions.
